File: MODUL-2/Python/PRAK202-2310817220002-RaudatulSholehah.py
Nilai1 = float(input("Masukkan Nilai Pertama : "))
Nilai2 = float(input("Masukkan Nilai Pertama : "))

hasil = Nilai1 + Nilai2

Nilai1=(f"{int(Nilai1)}" if Nilai1.is_integer() else f"{Nilai1:.2f}")

print(f'Hasil dari penjumlahan nilai pertama "{Nilai1}" dan nilai kedua "{Nilai2}" adalah "{hasil:.2f}"')

File: MODUL-2/C/PRAK202-2310817220002-RaudatulSholehah.c
#include <stdio.h>
#include <string.h>
int main()
{
    float Nilai1, Nilai2, hasil;
    printf("Masukkan Nilai Pertama      : ");
    scanf("%f", &Nilai1);

    printf("Masukkan Nilai Pertama      : ");
    scanf("%f", &Nilai2);

    hasil = Nilai1 + Nilai2;

    printf("Hasil dari penjumlahan nilai pertama \"%g\" dan nilai kedua \"%g\" adalah %.2f\n", Nilai1, Nilai2, hasil);
    return 0;
}